docs/db-model.puml

```diff
@@ -26,10 +32,12 @@ entity "**repository_configurations**" {
   *""name"": //character varying(255) //
   ""versions"": //character varying(255)[] //
   *""arch"": //character varying(255) //
-  *""account_id"": //character varying(255) //
+  ""account_id"": //character varying(255) //
   *""org_id"": //character varying(255) //
   *""org_id"": //character varying(255) //
   *""repository_uuid"": //uuid [FK]//
+  ""gpg_key"": //text //
+  *""metadata_verification"": //boolean //
```

Comment:
I'm unsure if metadata_verification can be undefined as well (if the * is as suggested above).

Reply:
same as above, its required in the db schema, but defaults to false